apps/numgen.py

```diff
@@ -129,17 +129,24 @@ def setDB(self):
         """Sets the database to store the number."""
         self.dbName = self.generatorFrame.dbBox.currentText()
         self.viewerFrame.statusLabel.setText("database updated")
+        self.broadcastRequested.emit(
+            "logbus", 
+            f"Database to store is set as {self.dbName}." if self.dbName
+            else "Database to store is not selected."
+        )
 
     @pyqtSlot()
     def generateNumber(self):
         """Generates and shows a random number when the button is clicked."""
         # generate a random number
         num = generate()
         self.viewerFrame.numberLabel.setText(f"generated number: {num}")
+        self.broadcastRequested.emit("logbus", f"Generate a random number: {num}.")
         # save the generated number
         dbPath = self.dbs[self.dbName]
         is_save_success = write(os.path.join(dbPath, self.dbName), self.table, num)
         if is_save_success:
             self.viewerFrame.statusLabel.setText("number saved successfully")
+            self.broadcastRequested.emit("logbus", f"Save the generated number: {num}.")
```

Comment:
How about "Saved ..." or "Generated number saved"?

Reply:
I think "Generated number saved" is good!